tests/ethpm/test_package.py

```diff
@@ -20,14 +20,14 @@ def deployed_safe_math(safe_math_package, w3):
     return safe_math_package, tx_receipt.contractAddress
 
 
-def test_package_object_instantiates_with_a_web3_object(all_standalone_manifests, w3):
-    for manifest in all_standalone_manifests:
+def test_package_object_instantiates_with_a_web3_object(all_manifests, w3):
+    for manifest in all_manifests:
```

Comment:
we often try to avoid loops like this in tests in favor of either:

- parametrized fixtures
- parametrized tests

```python
@pytest.fixture(params=[1, 2, 3])
def fixture_with_params(request):
    return request.param

@pytest.mark.parmetrize('value', [1, 2, 3])
def test_that_is_parametrized(value):
    ...
```